docs/manual/templates.rst

```diff
@@ -155,6 +155,27 @@ Expressie                           Voorbeeld waarde                   Toelichti
 .. note:: Op dit moment krijgt u altijd de Nederlandse vertalingen/lokalisatie.
    Er is nog geen ondersteuning voor andere talen.
 
+Template tags
+-------------
+
+De volgende template tags kunnen ook worden gebruikt in opmaak componenten.
+
+**get_value**
+
+Geeft de mogelijkheid om een waarde uit een dictionary uit te halen.
+Bijvoorbeeld, als deze variabele in het formulier bestaat:
+
+.. code:: python
+
+   eenVariabel = {"optie 1": "waarde 1", "optie 2": "waarde 2"}
+
+Dan de template tag ``get_value``:
```

Comment:
```suggestion
Dan zal de template tag ``get_value``:
```

Reply:
Shall I also change the last part `geeft de waarde 'waarde 1' terug` to `de waarde 'waarde 1' teruggeven`? And maybe also put it on the same line :thinking:

Reply:
Ohh nvm you specified it below :smile: